ReactAndroid/src/main/java/com/facebook/react/views/view/ReactViewGroup.java

```diff
@@ -61,6 +62,7 @@
   private static final LayoutParams sDefaultLayoutParam = new ViewGroup.LayoutParams(0, 0);
   /* should only be used in {@link #updateClippingToRect} */
   private static final Rect sHelperRect = new Rect();
+  public @Nullable ArrayList<Integer> mDeleteMark = new ArrayList<Integer>();
```

Comment:
creating an empty ArrayList for every ReactViewGroup is quite expensive, specially because this is not going to be used in most of the cases
Also this variable is not @Nullable